lib/multitenancy_tools/schema_dumper.rb

```diff
@@ -21,9 +21,11 @@ module MultitenancyTools
   class SchemaDumper
     # @param database [String] database name
     # @param schema [String] schema name
-    def initialize(database, schema)
+    def initialize(database, schema, options = {})
```

Comment:
Maybe you could use keyword arguments like this:
```ruby
def initialize(database, schema, host: '', username: '')
```

But I'm not sure, you have to check what is the minimum ruby version to be supported. (and it will be a good ideia to set the `spec.required_ruby_version = '>= X'` in gemspec

Reply:
it requires ruby 2+ and for required keyword args, ruby 2.1+. but I'd rather keep them on the options hash as adding more optional parameters would raise the number of arguments the method is taking. wdyt?